jax/numpy/lax_numpy.py

```diff
@@ -2941,6 +2946,14 @@ def vander(x, N=None, increasing=False):
 ### Misc
 
 
+@_wraps(np.argwhere)
+def argwhere(a):
+  result = transpose(vstack(nonzero(a)))
```

Comment:
I'm curious if the `vstack` is needed, given the NumPy docs say "np.argwhere(a) is almost the same as np.transpose(np.nonzero(a)), but produces a result of the correct shape for a 0D array."

Reply:
nonzero returns a tuple of 1D arrays, which ``np.transpose`` silently converts to a 2D array. Jax returns an error without the explicit vstack.